spec/message.abnf

```diff
@@ -1,43 +1,46 @@
-message = pattern / complex-message
+message = simple-message / complex-message
 
-complex-message = "{{" [s] *(declaration [s]) body [s] "}}"
+simple-message = [simple-start pattern]
+simple-start = simple-start-char / text-escape / expression
+pattern = *(text-char / text-escape / expression)
 
-declaration = input-declaration / local-declaration
+complex-message = *(declaration [s]) complex-body [s]
+declaration = input-declaration / local-declaration / reserved-keyword
```

Comment:
`reserved-keyword` doesn't sound right (it suggests that it is only the keyword). Perhaps `reserved-statement` or even just `reserved`. The corresponding (and different) `reserved-annotation` isn't in conflict with it.

Reply:
I'll rename as `reserved-statement`.